src/main/java/ca/corefacility/bioinformatics/irida/ria/web/models/export/NcbiSubmissionSample.java

```diff
@@ -0,0 +1,95 @@
+package ca.corefacility.bioinformatics.irida.ria.web.models.export;
+
+import java.util.List;
+
+import ca.corefacility.bioinformatics.irida.model.export.NcbiInstrumentModel;
+import ca.corefacility.bioinformatics.irida.model.export.NcbiLibrarySelection;
+import ca.corefacility.bioinformatics.irida.model.export.NcbiLibrarySource;
+import ca.corefacility.bioinformatics.irida.model.export.NcbiLibraryStrategy;
+
+/**
+ * Represents a sample in a {@link NcbiSubmissionRequest}
+ */
+public class NcbiSubmissionSample {
+	private String bioSample;
```

Comment:
Similar comment as above..this is the bio sample name correct? If so let's rename this for clarity

Reply:
See above response